src/main/java/org/opensearch/security/DefaultObjectMapper.java

```diff
@@ -287,12 +287,26 @@ public static TypeFactory getTypeFactory() {
         return objectMapper.getTypeFactory();
     }
 
+    @SuppressWarnings("removal")
```

Comment:
Is the change in this class used anywhere?

Reply:
its causing warnings without this suppress.